docs/docs/how-tos/nebari-local.md

```diff
@@ -159,6 +159,15 @@ security:
         tag: sha-b4a2d1e
 ```
 
+### Increase fs watches
+
+Depending on your host system, you may need to increase the `fs.inotify.max_user_watches` and
+`fs.inotify.max_user_instances kernel parameters` if you see a `too many open files` in the logs of
+a failing pod.
+
```

Comment:
Can you also add the fixer snippet that you listed in the issue? I think that would be beneficial.

```
sudo sysctl fs.inotify.max_user_watches=524288      
sudo sysctl fs.inotify.max_user_instances=512
```